ui-v2/app/utils/acls-status.js

```diff
@@ -23,6 +27,12 @@ export default function(P = Promise) {
       [propName]: p
         .catch(function(e) {
           switch (e.errors[0].status) {
+            case '500':
+              if (e.errors[0].detail.indexOf(UNKNOWN_METHOD_ERROR) === 0) {
```

Comment:
i think you want `!== -1` so that middle-of-the-string matches work.  When proxying between datacenters you will accumulate extra `rpc error making call:` prefixes for each hop